source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -85,7 +85,7 @@ def _get_role(self):
 		try:
 			return normalizeStdName(self.pdDomNode.GetStdName())[0]
 		except (AttributeError, LookupError, COMError, TypeError):
-			pass
+			log.debugWarning("Could not get role for AcrobatNode using normalizeStdName", exc_info=True)

```

Comment:
Will this be noisy? Are regular `LookupError` or `AttributeError` exceptions expected?

Reply:
Hmm, good point. Do you want me to only log this for type errors? I'd say that this could still be noisy though. May be I should avoid raising the typeerror altogether.